tests/selenium/test_sizing.py

```diff
@@ -0,0 +1,270 @@
+                assert (
+                    test.driver.execute_script(
+                        "return getComputedStyle(document.querySelector('#{} .dash-spreadsheet-container')).display".format(
+                            variation["id"]
+                        )
+                    )
+                    == "none"
+                )
+            else:
+                cells_are_same_width(target, table)
+
+        test.driver.find_element_by_css_selector("#btn").click()
+
+
+def test_szng002_percentages_result_in_same_widths(test):
+    base_props = dict(
+        columns=[
+            {"id": "_", "name": ["_", "_", "_"]},
+            {
+                "id": "a",
+                "name": [
+                    "A-----------------VERY LONG",
+                    "A-----------------VERY LONG",
+                    "A-----------------VERY LONG",
+                ],
+            },
+            {
+                "id": "b",
+                "name": [
+                    "A-----------------VERY LONG",
+                    "A-----------------VERY LONG",
+                    "B-----------------VERY LONG",
+                ],
+            },
+            {
+                "id": "c",
+                "name": [
+                    "A-----------------VERY LONG",
+                    "B-----------------VERY LONG",
+                    "C-----------------VERY LONG",
+                ],
+            },
+        ],
+        data=[
+            {"_": 0, "a": 85, "b": 601, "c": 891},
+            {"_": 0, "a": 967, "b": 189, "c": 514},
+            {"_": 0, "a": 398, "b": 262, "c": 743},
+            {"_": 0, "a": 89, "b": 560, "c": 582},
+            {"_": 0, "a": 809, "b": 591, "c": 511},
+        ],
+        style_table=dict(width=500, minWidth=500, maxWidth=500, paddingBottom=10),
+        style_cell=dict(width="20%", minWidth="20%", maxWidth="20%"),
+    )
+
+    _fixed_columns = [dict(headers=True, data=1), dict(headers=True)]
+
+    _fixed_rows = [dict(headers=True, data=1), dict(headers=True)]
+
+    _merge_duplicate_headers = [True, False]
+
+    variations = []
+    i = 0
+    for fixed_columns in _fixed_columns:
+        for fixed_rows in _fixed_rows:
+            for merge_duplicate_headers in _merge_duplicate_headers:
+                variations.append(
+                    {
+                        **base_props,
+                        "fixed_columns": fixed_columns,
+                        "fixed_rows": fixed_rows,
+                        "merge_duplicate_headers": merge_duplicate_headers,
+                        "id": "table{}".format(i),
+                    }
+                )
+                i = i + 1
+
+    tables = [DataTable(**variation) for i, variation in enumerate(variations)]
+
+    app = dash.Dash(__name__)
+    app.layout = Div(tables)
+
+    test.start_server(app)
+
+    target = test.driver.find_element_by_css_selector("#table0")
+
+    for i in range(1, len(variations)):
```

Comment:
Test various configurations of fixed columns/rows and merge_duplicate_headers to make sure they all render the same with % based column widths.